modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -109,9 +109,9 @@ xref:setting-up-mobile-services-to-demonstrate-showcase-apps[Setting up Mobile S
 		}
 	}]
 }
-
-module.exports = config;
 ----
++
+. Copy the content to your clipboard.
 
 [[configuring-the-showcase-app-to-point-at-the-provisioned-data-sync-service]]
 === Configuring the Showcase App to point at the provisioned Data Sync service
```

Comment:
@finp  - any proposals (https://github.com/aerogear/mobile-docs/pull/682#discussion_r291576080)? I can catch you at the stand-up tomorrow too if you want to talk it out...just let me know :+1: